src/test/kotlin/com/terraformation/backend/accelerator/db/ParticipantProjectSpeciesServiceTest.kt

```diff
@@ -160,6 +170,31 @@ class ParticipantProjectSpeciesServiceTest : DatabaseTest(), RunsAsUser {
                   projectId = projectId2,
                   submissionStatusId = SubmissionStatus.NotSubmitted)),
           submissionsDao.fetchByDeliverableId(deliverableId).map { it.copy(id = null) })
+
+      eventPublisher.assertEventPublished(
```

Comment:
There is also `TestEventPublisher.assertEventsPublished()` that takes a set of events to verify.